AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/EventManagerInviteController.cs

```diff
@@ -0,0 +1,114 @@
+﻿using AllReady.Areas.Admin.Features.EventManagerInvites;
+using AllReady.Areas.Admin.ViewModels.ManagerInvite;
+using AllReady.Features.Events;
+using AllReady.Models;
+using AllReady.Security;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+using System.Linq;
+
+namespace AllReady.Areas.Admin.Controllers
+{
+    [Area("Admin")]
+    [Authorize(nameof(UserType.OrgAdmin))]
+    public class EventManagerInviteController : Controller
+    {
+        private readonly IMediator _mediator;
+        private readonly UserManager<ApplicationUser> _userManager;
+
+        public EventManagerInviteController(IMediator mediator, UserManager<ApplicationUser> userManager)
+        {
+            _mediator = mediator;
+            _userManager = userManager;
+        }
+
+        // GET: Admin/Invite/Send/5
+        [Route("[area]/[controller]/[action]/{eventId:int}")]
+        public async Task<IActionResult> Send(int eventId)
+        {
+            var @event = await _mediator.SendAsync(new EventByEventIdQuery { EventId = eventId });
```

Comment:
Sounds like a good idea. Just a question, what do we want to do with the IsOrganizationAdmin check on the line after that uses the oranization ID. Should I add the organization ID to the ViewModel even if it is not used in the view or create another query just to get the oranization ID even if this would require another trip to the database?